src/test/kotlin/no/nav/tilleggsstonader/sak/util/SøknadBarnetilsynUtil.kt

```diff
@@ -67,8 +67,8 @@ object SøknadBarnetilsynUtil {
     )
 
     private fun lagHovedytelse(
-        hovedytelse: Hovedytelse = Hovedytelse.AAP,
+        vararg hovedytelse: Hovedytelse = arrayOf(Hovedytelse.AAP),
```

Comment:
hva er `vararg`? :O

Reply:
Den gjør att du kan bruke `lagHovedytelse(ytelse, ytelse2, ytelse3)` i stedet for å bruke en liste `lagHovedytelse(listOf(ytelse1, ytelse2, ytelse3))`
https://kotlinlang.org/docs/functions.html#variable-number-of-arguments-varargs